src/go/configinfo/service_info.go

```diff
@@ -719,6 +719,17 @@ func (s *ServiceInfo) processLocalBackendOperations() error {
 }
 
 func (s *ServiceInfo) processUsageRule() error {
+
+	// TODO(dafudeng) b/238652680
+	// hard code to allow health check, if defined, to bypass service control,
+	// need to refactor this.
+	if method, err := s.getMethod("grpc.health.v1.Health.Check"); err == nil {
+		method.SkipServiceControl = true
+	}
+	if method, err := s.getMethod("grpc.health.v1.Health.Watch"); err == nil {
```

Comment:
Coding wise, maybe to define a HardCodedSkipServiceControlMethodList  in "util", it is more flexible.

Reply:
Agree. Changed.